fluentui_controls/src/main/java/com/microsoft/fluentui/tokenized/controls/BasicCard.kt

```diff
@@ -26,21 +26,22 @@ import com.microsoft.fluentui.theme.token.controlTokens.CardType
 fun BasicCard(
     modifier: Modifier = Modifier,
     basicCardTokens: BasicCardTokens? = null,
+    basicCardInfo: BasicCardInfo? = null,
     content: @Composable () -> Unit
 ) {
     val themeID =
         FluentTheme.themeID    //Adding This only for recomposition in case of Token Updates. Unused otherwise.
     val token = basicCardTokens
         ?: FluentTheme.controlTokens.tokens[ControlTokens.ControlType.BasicCardControlType] as BasicCardTokens
-    val basicCardInfo = BasicCardInfo(CardType.Elevated)
```

Comment:
Added another option in CardType.NonElevated & expose this variable in BasicCard api instead of BasicCardInfo.